constants.xml

```diff
@@ -136,7 +136,8 @@
     -->
     </lookups>
     <gpu>
-        <NUM_DEVICE_TO_USE>1</NUM_DEVICE_TO_USE>
+        <!-- Which GPU to use, can use multile GPU, for example: 0, 2, 4 -->
```

Comment:
Can you revise to `Which GPU to use, can use multiple GPU, for example: "0, 2, 4"`
(i.e., Fixed the misspelled "multiple" and added quotes in the example to make it clear it is the actual value you want to put in this field)

Reply:
Ok